website/common/script/content/hatching-potions.js

```diff
@@ -49,44 +49,44 @@ let drops = {
 };
 
 let premium = {
-  Spooky: {
```

Comment:
Looks like whitespacing got adjusted by accident--it doesn't have a functional impact, but the standard in our code is two spaces per indent.